CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [0.2.1] - 2019-11-17
+
+### Added
+
+- Added new `$in_list` parameter to the `PrefixAllGlobalsSniff::process_variable_assignment()` method, so that the method is compatible with the upstream WPCS method (changed in WPCS 2.2.0).
```

Comment:
Just thinking - this could probably be moved to the `Changed` section too as it's not really something added to TRTCS, just a minor change in the function signature.